templates/company/index.html

```diff
@@ -0,0 +1,173 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}{% endblock %}
+
+{% block title %}Company{% endblock %}
+
+{% block content %}
+
+<section class="p-strip--suru-light-grey">
+  <div class="u-fixed-width">
+    <h1>Delivering secure, high-quality <br> open source</h1>
+    <p>We empower anyone, anywhere, to access the best of open source <br> technologies</p>
+    <p><a href="" class="p-button--positive">Get in touch</a></p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>About Canonical</h2>
+  </div>
+  <div class="row">
+    <div class="col-4 u-equal-height">
+      <div class="p-card--highlighted has-accent-alt-border">
+        <h3 class="p-heading--4">Careers</h3>
```

Comment:
All Vanilla examples have simply H3 no class here.

Reply:
This is to match the design in Miro